source/include/Events.h

```diff
@@ -77,21 +77,23 @@ struct EventCommonData {
     static UInt32 GetNumberOfCommonElements() { return 5; }  // change if fields are added above
 
     void InitEventTime() { eventTime = UInt32(gPlatform.Timer.TickCountToMilliseconds(gPlatform.Timer.TickCount())); }
-    EventCommonData(UInt32 source, UInt32 type) : eventSource(source), eventType(type), eventSeqIndex(0) {
+
+    EventCommonData(const UInt32 source, const UInt32 type) : eventSource(source), eventType(type), eventSeqIndex(0) {
```

Comment:
> UInt32 source ==> const UInt32 source

I can't seem to find a clang-tidy rule that enforces this. Did you find one?

Reply:
remove const modifiers.